sys/include/net/gnrc/rpl.h

```diff
@@ -82,6 +82,42 @@
  *   CFLAGS += -DCONFIG_GNRC_RPL_WITHOUT_VALIDATION
  *   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  *
+ * - This RPL implementation currently only supports storing mode.
+ *   That means, in order to have downwards routes to all nodes the
+ *   Neighbor Information Base must be big enough to store information for
+ *   each node.
+ *   To store the actual routes, the number of off-link entries in NIB
```

Comment:
What are off-link entries?

Reply:
Too much implementation detail on the NIB IMHO. This sentence can be dropped. That two configuration parameters need to be changed should be clear from the rest. The interested reader can then look at the NIB itself, to find out why that is ;-).